java-lib/src/main/java/com/wavefront/data/Validation.java

```diff
@@ -47,9 +48,11 @@
 
   private final static LoadingCache<String, Counter> ERROR_COUNTERS = Caffeine.newBuilder().
       build(x -> Metrics.newCounter(new MetricName("point", "", x)));
+  private final static LoadingCache<String, Counter> LOG_ERROR_COUNTERS = Caffeine.newBuilder().
+          build(x -> Metrics.newCounter(new MetricName("log", "", x)));
```

Comment:
This is over kill, we can use [map.computeIfAbsent](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-) for this

Reply:
and lower case ;-)

Reply:
chatted offline German said he would refactor it later.  I'm just pattern matching and doing what we already do here :)